src/importexport/musicxml/internal/musicxml/musicxml.h

```diff
@@ -23,122 +23,9 @@
 #ifndef __MUSICXML_H__
 #define __MUSICXML_H__
 
-/**
- \file
- Definition of class MusicXML
-*/
-
-#include "engraving/types/fraction.h"
-#include "engraving/dom/mscore.h"
-#include "engraving/dom/pitchspelling.h"
-#include "engraving/dom/line.h"
-#include "importxmlfirstpass.h"
-#include "musicxmlsupport.h"
-
 namespace mu::engraving {
-//---------------------------------------------------------
-//   MusicXmlPartGroup
-//---------------------------------------------------------
-
-struct MusicXmlPartGroup {
-    int span = 0;
-    int start = 0;
-    BracketType type = BracketType::NO_BRACKET;
-    bool barlineSpan = false;
-    int column = 0;
-};
-
 const int MAX_LYRICS       = 16;
 const int MAX_PART_GROUPS  = 8;
```

Comment:
These two are used in only one file each. Are they very general musicxml constants, or would it make sense to move them to those respective files?
If the latter, then it would be a bit sad if `MAX_NUMBER_LEVEL` becomes the only one that's left, so in that case I'd move that to musicxmltypes.h (even though it's a constant and not a type... but we have that in other types.h files too).